src/allmydata/test/web/test_web.py

```diff
@@ -2206,11 +2226,14 @@ def test_POST_DIRURL_deepsize(self):
         def _got_json(data):
             self.failUnlessReallyEqual(data["finished"], True)
             size = data["size"]
-            self.failUnless(size > 1000)
+            # Match calculation of text value size below:
+            self.failUnless(
+                size.get("size-directories", 0) + size.get("size-mutable-files", 0) +
+                size.get("size-immutable-files", 0) > 1000)
```

Comment:
was this a non-deterministic test?  or does it share state with some other test?  or is this just a free enhancement? :)

Reply:
The test was previously broken, asserting something like assertTrue(str(json["size"])) and actually doing str of a dict...